### PR TITLE
Connect navigation

### PR DESCRIPTION
**Pull request by oscaregr**
connection the views than no have one correct navigation, in admin interface

# Description

What did you implemented/Why did you implemented this:

Examples:

- What?: i've connected the views for made a correct navigation
- Why?: because the admin interface need this

## Screenshots
image
image

### CODE REVIEW

**Review by AangelF — APPROVED** [review]

**Review by Reiyirak — APPROVED** [review]
LGTM

File: app/views/companies/index.html.erb [hunk 1]
```diff
@@ -1,7 +1,11 @@
+<a href="<%= admin_index_path %>" class="a">
+  <button class="back">Volver</button>
```
Comment (juperameza) [inline app/views/companies/index.html.erb:2]: Here

File: app/views/employees/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<a href="<%= admin_index_path %>" class="a">
+  <button class="back">Volver</button>
```
Comment (juperameza) [inline app/views/employees/index.html.erb:2]: Here

**Review by juperameza — CHANGES_REQUESTED** [review]
Everything looks good but choose a language don't mix Spanish and English and since everything seems to be on English I recommend you change those "volver" to bsck

**Review by juperameza — APPROVED** [review]
Lgtv 😼

File: app/views/employees/index.html.erb (hunk 2 above)
Comment (oscaregr) [reply app/views/employees/index.html.erb]: done

File: app/views/companies/index.html.erb (hunk 1 above)
Comment (oscaregr) [reply app/views/companies/index.html.erb]: done

**Pull request merged**